package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@datadog/pprof",
-  "version": "1.0.0",
+  "version": "1.1.2",
```

Comment:
The existing process is to only change version in proposal PRs to vN.x branches, so main would always be N.0.0 where N is the latest major line. Not sure if/how this would impact your plan to dev-release every commit.

@rochdev Any thoughts on if/how this might matter? Is it reasonable with our existing CI release process to change version in main? Could also just do all these dev releases like `N.0.0-prerelease.0`, though the version differentiation _would_ be kind of nice. 🤔

Reply:
This shouldn't be done in `main`. There could be a new branch however for these releases that pulls from `main`, changes the version number, and publishes.

Reply:
I changed the version in main to `2.0.0-pre` (as it's done in [dd-trace-js](https://github.com/DataDog/dd-trace-js/blob/master/package.json#L3)).
CI, when run on main branch, will publish unique versions like: `2.0.0-pre-197d240+3751835979.1` with `dev` tag. Version number will not change in `package.json` on main branch.
Would that be OK ?

Or do you prefer that I follow the approach suggested by @rochdev? (if I understand correctly, the idea is to create a branch dedicated to these releases, when a commit is pushed on main, a github action would propagate this commit on the dedicated branch, increment and commit the new version and then publish with `dev` tag)

Reply:
I'm fine with not tracking minor/patch exactly on dev releases. What about you, @rochdev?

Reply:
In the current scheme, commit sha is embedded in the beta version ( beta version is built as`{current_version}-{short_sha}-{ci_run_id}.{ci_run_attempt}`).
@rochdev @Qard Is that OK for you ?